src/zampy/datasets/land_cover.py

```diff
@@ -0,0 +1,249 @@
+
+    bib = """
+    @article{buchhorn2020copernicus,
+    title={Copernicus global land cover layers—collection 2},
+    author={Buchhorn, Marcel et al.},
+    journal={Remote Sensing},
+    volume={12},
+    number={6},
+    pages={1044},
+    year={2020},
+    publisher={MDPI}
+    }
+    """
+
+    data_url = "https://cds.climate.copernicus.eu/cdsapp#!/dataset/satellite-land-cover?tab=overview"
+
+    cds_dataset = "satellite-land-cover"
+
+    def __init__(self) -> None:
+        """Init."""
+        pass
+
+    def download(
+        self,
+        download_dir: Path,
+        time_bounds: TimeBounds,
+        spatial_bounds: SpatialBounds,
+        variable_names: list[str],
+        overwrite: bool = False,
+    ) -> bool:
+        validation.validate_download_request(
+            self,
+            download_dir,
+            time_bounds,
+            spatial_bounds,
+            variable_names,
+        )
+
+        download_folder = download_dir / self.name
+        download_folder.mkdir(parents=True, exist_ok=True)
+
+        cds_utils.cds_request_land_cover(
+            dataset=self.cds_dataset,
+            time_bounds=time_bounds,
+            path=download_folder,
+            overwrite=overwrite,
+        )
+
+        write_properties_file(
+            download_folder, spatial_bounds, time_bounds, variable_names
+        )
+
+        return True
+
+    def ingest(
+        self,
+        download_dir: Path,
+        ingest_dir: Path,
+        overwrite: bool = False,
+    ) -> bool:
+        download_folder = download_dir / self.name
+        ingest_folder = ingest_dir / self.name
+        ingest_folder.mkdir(parents=True, exist_ok=True)
+
+        archive_file_pattern = f"{self.name}_*.zip"
+        archive_files = list(download_folder.glob(archive_file_pattern))
+
+        for file in archive_files:
+            unzip_raw_to_netcdf(
+                ingest_folder,
+                file=file,
+                overwrite=overwrite,
+            )
+
+        copy_properties_file(download_folder, ingest_folder)
+
+        return True
+
+    def load(
+        self,
+        ingest_dir: Path,
+        time_bounds: TimeBounds,
+        spatial_bounds: SpatialBounds,
+        resolution: float,
+        regrid_method: str,  # Unused in land-cover dataset
+        variable_names: list[str],
+    ) -> xr.Dataset:
+        files: list[Path] = []
+        for var in variable_names:
+            if var not in self.variable_names:
+                msg = (
+                    "One or more variables are not in this dataset.\n"
+                    f"Please check input. Dataset: '{self.name}'\n"
+                    f"Variables: '{variable_names}'"
+                )
+                raise ValueError(msg)
+        files = list((ingest_dir / self.name).glob(f"{self.name}_*.nc"))
+
+        ds = xr.open_mfdataset(files, chunks={"latitude": 200, "longitude": 200})
```

Comment:
Is it possible to make the chunk size configurable like an argument in `load()` with default values `(200, 200)`?

Reply:
We could at some point decide to add a dask configuration file where you specify the desired chunk size in MiB, and use that to determine chunk sizing. But I think it's a bit too much work to do so now.

Reply:
Thanks for your review. We can add it as an argument. But the philosophy is, our users are not supposed to see this function/API, or configure anything themselves. They must use the recipe to download all the datasets that we support in zampy (https://github.com/EcoExtreML/zampy/blob/main/docs/using_zampy.md). Therefore, when coding this part we simply choose the optimal parameters for the user.

It depends on the pathway of the design. If we do expect users to call these functions and download stuff themselves in a notebook/ipython environment, then we do need to make them configurable. Let's have a discussion about it.

Reply:
An issue created for this #37.